src/rebar_relx.erl

```diff
@@ -72,13 +71,42 @@ do(Provider, State) ->
 
     {ok, State}.
 
+read_relx_config(State, Options) ->
+    ConfigFile = proplists:get_value(config, Options, []),
+    case ConfigFile of
+        "" ->
+            case {rebar_state:get(State, relx, []), file:consult("relx.config")} of
```

Comment:
We may want to use `filename:join([rebar_dir:root_dir(State), "relx.config"])` since we allow changing the base context of where things run.

Reply:
Should this also affect `--config` flag? I didn't implement that.

Reply:
Fixed in https://github.com/erlang/rebar3/pull/2435/commits/7884a4b109af1d11dcf887e391a99f189aaf165e